easybuild/easyconfigs/c/cURL/cURL-7.58.0-intel-2018a.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'ConfigureMake'
+
+name = 'cURL'
+version = '7.58.0'
+
+homepage = 'http://curl.haxx.se'
+
+description = """
+ libcurl is a free and easy-to-use client-side URL transfer library,
+ supporting DICT, FILE, FTP, FTPS, Gopher, HTTP, HTTPS, IMAP, IMAPS, LDAP,
+ LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, Telnet and TFTP.
+ libcurl supports SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP
+ form based upload, proxies, cookies, user+password authentication (Basic,
+ Digest, NTLM, Negotiate, Kerberos), file transfer resume, http proxy tunneling
+ and more.
+"""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
```

Comment:
@tutufan We usually build `cairo` with `GCCcore`, since that way it's compatible with both `foss` and `intel`.

We already have a `cURL-7.58.0-GCCcore-6.4.0.eb` easyconfig in `develop`, so you can just remove this one?